README.md

```diff
@@ -14,12 +14,12 @@ environment.
 
 From inside this project's git repo, run the following command:
 
-    # docker build -t ceph-dev-docker .
+    $ docker build -t ceph-dev-docker .
```

Comment:
You have to execute docker as root, so i think this should be visualized by using the # prompt, otherwise use ``$ sudo docker ...``.

Reply:
I agree with @votdev